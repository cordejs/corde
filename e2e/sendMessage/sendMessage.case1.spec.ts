import { runCLI } from "../cliRunner";

import Utils from "../testUtils";

it("should return the sent message", async () => {
  const command = Utils.buildCommandWithConfigPath("sendMessage", "bot_case1.spec.ts");
  const [mockProcess, stdout] = await runCLI(command);
  expect(mockProcess).toBeCalledWith(0);
  expect(stdout).toMatchSnapshot();
});
