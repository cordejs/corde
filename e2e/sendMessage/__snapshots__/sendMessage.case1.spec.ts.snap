// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the sent message 1`] = `
"node test --config ./e2e/corde.config.ts --files ./e2e/sendMessage/__cordeTest__/bot_case1.spec.ts
TEST MESSAGE

 PASS   e2e\\\\sendMessage\\\\__cordeTest__\\\\bot_case1.spec.ts   100ms
   âœ”  should send a message   100ms

Test Files: 1 passed. 1 total
Tests:      1 passed. 1 total
Time:       100ms
"
`;
